core/templates/services/exploration-html-formatter.service.spec.ts

```diff
@@ -40,16 +40,16 @@ describe('Exploration Html Formatter Service', () => {
 
   it('should correctly set interaction HTML for TextInput when it is in' +
      ' editor mode', () => {
-    var interactionId = 'TextInput';
+    var interactionId = 'FractionsInput';
```

Comment:
Why this change? Any specific reason? [We don't expect any changes in test file until the whole functionality is changed]

Note: the test looks `TextInput` specific as per the test message o line 41.

Reply:
The functionality is changed, I need to change this cover the file completely. It would have been better if all the interactions were tested in this spec but it is not the case. So I changed it. Changing the test description.